package.json

```diff
@@ -13,12 +13,15 @@
     "cache:clear": "rm -rf .cache",
     "evals": "npm run build-dom-scripts && tsx evals/index.eval.ts",
     "build-dom-scripts": "tsx lib/dom/genDomScripts.ts",
-    "build": "npm run build-dom-scripts && tsup lib/index.ts --dts",
-    "release": "changeset publish"
+    "build-types": "tsc --emitDeclarationOnly --outDir dist",
+    "build-js": "esbuild lib/index.ts --bundle --platform=node --outdir=dist --format=esm --minify --tree-shaking=true --packages=external",
```

Comment:
the `--packages=external` piece doesn't seem to be working as intended. i'm seeing the whole browserbasehq sdk included in the dist. may have to do with relative imports or something similar. maybe we want `--external=*`. the intention is to exclude dependencies but bundle the files here right?